sonar-helm-for-iac/helm-imports.go

```diff
@@ -77,15 +86,20 @@ var sprigFunctionsWhitelist = []string{
 	"first",
 	"float64",
 	"floor",
+	"fromJson",
```

Comment:
Including this would mean a duplicate of the method, as we already manually implemented this.

Reply:
Good catch! It would be overridden by the latest implementation, but I'll still removed it for consistency.